.eslintrc.json

```diff
@@ -1,116 +1,68 @@
 {
-    "env": {
-        "browser": true,
-        "es6": true
-    },
-    "extends": [
-      "eslint:recommended",
-      "plugin:react/recommended",
-      "plugin:react-hooks/recommended"
+  "env": {
+    "browser": true,
+    "es6": true,
+    "jest": true,
+    "node": true
+  },
+  "plugins": ["prettier"],
```

Comment:
According to the `eslint-plugin-prettier` doc, we should be able to get the same config in only one step using the recommended config : `This plugin ships with a plugin:prettier/recommended config that sets up both the plugin and eslint-config-prettier in one go.`

Some needed rules like "prettier/prettier": "error" would be automatically added a few eslint core rules which could cause issues would "automatically" be turned off.

Given these reasons, do you think we should use `plugin:prettier/recommended` instead ?

Reply:
Oh, I took inspiration from the Phoenix config and didn't see this option.
It doesn't do much but it may be a good idea indeed, to simplify a bit and guard against future ESLint stuff that may conflict.
I'll give it a go and see if all works the same :+1: